src/panels/lovelace/hui-editor.ts

```diff
@@ -60,6 +61,7 @@ class LovelaceFullConfigEditor extends LitElement {
                 class="save-button
```

Comment:
To do this correctly, we should just put this _inside_ the `mwc-button` tag. Solves RTL naturally

Reply:
Was already in there... We decided to move to text. Done